arcgis-ios-sdk-samples/Content Display Logic/Model/Sample.swift

```diff
@@ -14,11 +14,19 @@
 // limitations under the License.
 //
 
-struct Sample: Equatable {
+struct Sample: Hashable {
+    
     var name: String
     var description: String
     var storyboardName: String
     var dependencies: [String]
+    var readmeURL:URL?{
+        return Bundle.main.url(forResource: "README", withExtension: "md", subdirectory: name)
+    }
+    
+    var hashValue: Int{
```

Comment:
Was the compiler-provided implementation not working for you?

Reply:
It wasn't at the time for some reason but it is now.

Reply:
With the introduction of Swift 4.2 `Hashable` conformance is now done automatically.

https://github.com/apple/swift-evolution/blob/master/proposals/0206-hashable-enhancements.md